### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Robertorosmaninho**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,17 @@ TL;DR build taco using CMake. Run `make test`.
...
 
-Build taco using CMake 3.4.0 or greater:
+## Dependencies
+- CMake 2.8.12 or greater
+- GNU Make 4.2.1 or greater
+- conda 4.8.2 or greater
+
+## Create and activate conda environment:
+
+    conda env create -f .conda/environment.yml
```
Comment (guilhermeleobas) [inline README.md]:
```suggestion
    conda env create -f /path/to/taco/.conda/environment.yml
```

File: README.md [hunk 2]
```diff
@@ -29,7 +29,17 @@ TL;DR build taco using CMake. Run `make test`.
 # Build and test
 ![Build and Test](https://github.com/RSenApps/taco/workflows/Build%20and%20Test/badge.svg?branch=master)
 
-Build taco using CMake 3.4.0 or greater:
+## Dependencies
+- CMake 2.8.12 or greater
+- GNU Make 4.2.1 or greater
+- conda 4.8.2 or greater
```
Comment (guilhermeleobas) [inline README.md]: Given we use conda to download the dependencies, you don't need to specify them here. Maybe just specify conda and the LLVM version we're using (LLVM 11)

**Review by guilhermeleobas — CHANGES_REQUESTED** [review]
LGTM, just a couple of small changes

File: README.md (hunk 1 above)
Comment (Robertorosmaninho) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Robertorosmaninho) [reply README.md]: Done

**Review by guilhermeleobas — APPROVED** [review]

**Pull request merged**